### PR TITLE
vorbis 1.3.6

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **vorbis/1.3.6**

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
@danimtb Could you check Jenkins please? Here out status is NOK (red), but if you check Jenkins PRs, the #462 has only one build and it's OK (green). Seems like it didn't trigger a new build after the commit 118b515 (latest).

**Comment by danimtb** [comment]
Could be related to #479

File: recipes/vorbis/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            else:
+                self.copy(pattern="*.a", dst="lib", keep_path=False)
+        for pdb_file in glob.glob(os.path.join(self.package_folder, "bin", "*.pdb")):
+            os.unlink(pdb_file)
+
+    def package_info(self):
+        self.cpp_info.libs = ['vorbisfile', 'vorbisenc', 'vorbis']
+        if self.settings.os == "Linux":
+            self.cpp_info.system_libs.append("m")
+        self.cpp_info.name = 'VORBIS'
```
Comment (SSE4) [inline recipes/vorbis/all/conanfile.py]:
```suggestion
        self.cpp_info.names["cmake_find_package"] = 'VORBIS'
        self.cpp_info.names["cmake_find_package_multi"] = 'VORBIS'
```

File: recipes/vorbis/all/test_package/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
+import os
+import subprocess
+import sys
```
Comment (danimtb) [inline recipes/vorbis/all/test_package/conanfile.py]: Imports not needed

**Pull request merged**